lib/secp256k1/privatekey.ex

```diff
@@ -0,0 +1,153 @@
+defmodule Bitcoinex.Secp256k1.PrivateKey do
+  @moduledoc """
+   	Contains an integer used to create a Point and sign.
+  """
+  alias Bitcoinex.Secp256k1.{Params, Math, Point, Signature}
+  alias Bitcoinex.Base58
+
+  @type t :: %__MODULE__{
+          s: integer()
```

Comment:
d is the standard notation for private key integer

Reply:
accepted